src/Plug.vala

```diff
@@ -30,7 +30,7 @@ public class PantheonShell.Plug : Switchboard.Plug {
         settings.set ("desktop/appearance/wallpaper", "wallpaper");
         settings.set ("desktop/appearance", "appearance");
         settings.set ("desktop/dock", "dock");
-        settings.set ("desktop/hot-corners", "hotc");
+        settings.set ("desktop/hot-corners", "multitasking");
```

Comment:
This is the one thing I had a question about. Not sure how to handle it.

Reply:
I don't think this is user-facing in any way so it's probably safe to support both. But "hotc" is so obviously bad that if we didn't fix that it must mean we never link to it

Reply:
"hotc" was just the name of the stack child, but yeah my question was do we keep this, add multitasking, or what? I'll add the other and note this one as deprecated in a comment like we've done elsewhere.

Reply:
OH derp I read this wrong. Yes, I would say keep `"desktop/hot-corners"` in case someone is using this and then also add `"desktop/multitasking"